app/public/technologies.json

```diff
@@ -1,14 +1,317 @@
 {
```

Comment:
Should we add 
Cloud Conformity? - Trial or Adopt
Docker? - Adopt
BDD - Adopt
Sumo Logic needs to be added to our website (train of thought)
.Net - Hold
Kubernetes - Adopt in limited circumstances
Cucumber - Adopt

Reply:
Yes!